### PR TITLE
Improving install

### PR DESCRIPTION
**Pull request by MichaelJamesStam**
Made DE-STRESS a lot easier to install locally and also to run.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+EVOEF2_BINARY_PATH=/dependencies_for_de-stress/EvoEF2/EvoEF2
+DFIRE2_FOLDER_PATH=/dependencies_for_de-stress/DFIRE2-pair/
+ROSETTA_BINARY_PATH=/dependencies_for_de-stress/rosetta/source/bin/score_jd2.linuxgccrelease
+AGGRESCAN3D_SCRIPT_PATH=/dependencies_for_de-stress/Aggrescan3D/aggrescan3D_cli_run.py
+
+RQ_DASHBOARD_REDIS_URL=redis://redis:6379
+RQ_DASHBOARD_PORT=8182
+
+MAX_RUN_TIME=30
+HEADLESS_DESTRESS_WORKERS=3
```
Comment (ChrisWellsWood) [inline .env]: `.env` should not be in the code base, this one is for testing I think and there is already a file called .env-testing.

File: run_destress_headless.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+▓█████  ██▀███   ██▀███   ▒█████   ██▀███  
+▓█   ▀ ▓██ ▒ ██▒▓██ ▒ ██▒▒██▒  ██▒▓██ ▒ ██▒
+▒███   ▓██ ░▄█ ▒▓██ ░▄█ ▒▒██░  ██▒▓██ ░▄█ ▒
+▒▓█  ▄ ▒██▀▀█▄  ▒██▀▀█▄  ▒██   ██░▒██▀▀█▄  
+░▒████▒░██▓ ▒██▒░██▓ ▒██▒░ ████▓▒░░██▓ ▒██▒
+░░ ▒░ ░░ ▒▓ ░▒▓░░ ▒▓ ░▒▓░░ ▒░▒░▒░ ░ ▒▓ ░▒▓░
+ ░ ░  ░  ░▒ ░ ▒░  ░▒ ░ ▒░  ░ ▒ ▒░   ░▒ ░ ▒░
+   ░     ░░   ░   ░░   ░ ░ ░ ░ ▒    ░░   ░ 
+   ░  ░   ░        ░         ░ ░     ░     
+
```
Comment (ChrisWellsWood) [inline run_destress_headless.py:57]: I love this 😃

**Review by ChrisWellsWood — CHANGES_REQUESTED** [review]
Double check the .env file and remove it if it was a mistake. After that, happy to merge.

File: run_destress_headless.py (hunk 2 above)
Comment (MichaelJamesStam) [reply run_destress_headless.py:57]: This was actually @ESPhoenix 's idea.

File: .env (hunk 1 above)
Comment (MichaelJamesStam) [reply .env]: This has all been fixed now.

**Review by ChrisWellsWood — APPROVED** [review]
Looks good, merging.

**Pull request merged**